themes/tiddlywiki/vanilla/base.tid

```diff
@@ -83,7 +83,6 @@ h1, h2, h3, h4, h5, h6 {
 
 pre {
 	display: block;
-	padding: 14px;
```

Comment:
@pmario, while I agree that this PR is not the place to kind sneak this change into the core ;-), I'd also be in favor of using the the 0.5em padding of hljs, as I think code-samples should stand out a little. After all, that's why they're there.